RCS_client/RCS_client.py

```diff
@@ -0,0 +1,71 @@
+#usr/bin/env python3
+import rclpy
+import paho.mqtt.client as mqtt
+from rclpy.node import Node
+from std_msgs.msg import String
+
+class RCS_client(Node):
+    def __init__(self):
+        super().__init__("RCS_client")
```

Comment:
Look at main comment above.
```suggestion
        super().__init__("aks_rcs_client")
```